parser/tests/expr.rs

```diff
@@ -2,14 +2,15 @@ use ast::call::{Call, ProgrammaticCall, Redir, RedirFd, RedirOp, Redirected};
 use ast::control_flow::{For, ForKind, RangeFor};
 use ast::lambda::LambdaDef;
 use ast::operation::{BinaryOperation, BinaryOperator};
-use ast::r#type::{SimpleType, Type};
+use ast::r#type::{CastedExpr, SimpleType, Type};
 use ast::range::{Iterable, NumericRange};
 use ast::value::{Literal, LiteralValue};
 use ast::variable::{Assign, TypedVariable, VarDeclaration, VarKind, VarReference};
 use ast::Expr;
 use context::source::Source;
 use parser::parse;
 use pretty_assertions::assert_eq;
+use ast::group::{Parenthesis};
```

Comment:
```suggestion
use ast::group::Parenthesis;
```
This import will most likely be moved up by rustfmt.